app/javascript/components/analytics/widgets/ExposureSummary.js

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
 import { PropTypes } from 'prop-types';
-import { Card, Button, Row, Col } from 'react-bootstrap';
-import { ResponsiveContainer, BarChart, Bar, XAxis, YAxis, CartesianGrid, Tooltip, Legend } from 'recharts';
-import CustomizedAxisTick from './CustomizedAxisTick';
+import { Button, Card, Col, Row } from 'react-bootstrap';
+import { Bar, BarChart, CartesianGrid, Legend, ResponsiveContainer, Tooltip, XAxis, YAxis } from 'recharts';
+
+import { mapToChartFormat, parseOutFields } from '../../../lib/AnalyticsUtils';
```

Comment:
Only added `import { mapToChartFormat, parseOutFields } from '../../../lib/AnalyticsUtils';`. Everything else here is just moving `imports` around / alphabetizing them.